Sources/NumericText/NumericTextModifier.swift

```diff
@@ -17,10 +19,12 @@ public struct NumericTextModifier: ViewModifier {
     ///   - text: The string that this should observe and filter
     ///   - number: A number that should be updated whenever the `text` is updated
     ///   - isDecimalAllowed: Should the user be allowed to enter a decimal number, or an integer
-    public init(text: Binding<String>, number: Binding<NSNumber?>, isDecimalAllowed: Bool) {
+    ///   - numberFormatter: Custom number formatter used for formatting number in view
+    public init(text: Binding<String>, number: Binding<NSNumber?>, isDecimalAllowed: Bool, numberFormatter: NumberFormatter? = nil) {
```

Comment:
```suggestion
    public init(text: Binding<String>, number: Binding<NSNumber?>, isDecimalAllowed: Bool, numberFormatter: NumberFormatter = decimalNumberFormatter) {
```

Reply:
There is one disadvantage of that solution, we have to do `decimalNumberFormatter` public (now it's internal). Otherwise, we have a Xcode error: `Var 'decimalNumberFormatter' is internal and cannot be referenced from a default argument value`. I'm not sure that it's a good idea?

Reply:
Ah, good point.